src/components/FormGroup/SbFormGroup.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <fieldset class="sb-form-group" :class="componentClasses">
+    <div class="sb-form-group__info">
+      <legend class="sb-form-group__legend">{{ legend }}</legend>
+      <p v-if="description" class="sb-form-group__description">
+        {{ description }}
+      </p>
+    </div>
+    <div v-if="inline" class="sb-form-group__slot">
+      <slot />
+    </div>
+    <slot v-else />
+  </fieldset>
+</template>
+
+<script>
+export default {
+  name: 'SbFormGroup',
+
+  props: {
+    inline: {
```

Comment:
Wouldn't be a better approach to use `inline: Boolean` instead of having `type` and `default`?